devsetup/standalone/post_config/ironic.sh

```diff
@@ -16,33 +16,36 @@
 set -ex
 
 export OS_CLOUD="standalone"
-IRONIC_PYTHON_AGENT_URI=${IRONIC_PYTHON_AGENT_URI:-"https://images.rdoproject.org/centos9/master/rdo_trunk/current-tripleo/ironic-python-agent.tar"}
+IRONIC_PYTHON_AGENT_URI=${IRONIC_PYTHON_AGENT_URI:-"https://tarballs.opendev.org/openstack/ironic-python-agent/dib/ipa-centos8-stable-wallaby.tar.gz"}
```

Comment:
why not one of the centos9 images?

Reply:
ack, and I guess needs to be wallaby for standalone

Reply:
It might work with a later release, but the risk is that something changed in the API so a newer IPA may fail. Safer to use wallaby IPA on CS8. :+1: